test/core/client_channel/lb_policy/round_robin_test.cc

```diff
@@ -37,10 +35,10 @@ namespace {
 
 class RoundRobinTest : public LoadBalancingPolicyTest {
  protected:
-  RoundRobinTest() : lb_policy_(MakeLbPolicy("round_robin")) {}
+  RoundRobinTest() : LoadBalancingPolicyTest("round_robin") {}
 
   void ExpectStartup(absl::Span<const absl::string_view> addresses) {
```

Comment:
This function is no longer needed.  I thought I removed it in one of my commits, but maybe that got lost in a merge somewhere.

Reply:
let's clean it up separately?

Reply:
This PR is removing all callers of this function, so I think it should remove the function too.  There's no point leaving unused code lying around.

Reply:
didn't realize we were removing usage here... long week :)